test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/lang/StackTraceElementTest.scala

```diff
@@ -14,11 +14,26 @@ package org.scalajs.testsuite.javalib.lang
 
 import org.junit.Assert._
 import org.junit.Test
+import java.io._
+
+import org.scalajs.testcommon._
```

Comment:
This doesn't compile when doing `testSuiteJVM/test`. The correct place for tests of the test-common serializers is in https://github.com/scala-js/scala-js/blob/ccdc1f00d100a151457aabc86d1d7f05dafd9063/test-common/src/test/scala/org/scalajs/testcommon/SerializerTest.scala#L18.

Reply:
Fixed in https://github.com/scala-js/scala-js/pull/3631/commits/31bad02559e0a73cbf9e0cd486f1a6b3a235a011